src/Workspaces/Core/Portable/Workspace/Solution/SolutionState_Checksum.cs

```diff
@@ -112,17 +104,14 @@ private async Task<SolutionStateChecksums> ComputeChecksumsAsync(
             {
                 // get states by id order to have deterministic checksum.  Limit expensive computation to the
                 // requested set of projects if applicable.
```

Comment:
Comment probably needs revision to say we're using the already-in-order state.

Reply:
Done!